client/data/authorizations/action-types.ts

```diff
@@ -2,6 +2,7 @@
 
 export default {
 	SET_AUTHORIZATIONS: 'SET_AUTHORIZATIONS',
+	SET_AUTHORIZATION: 'SET_AUTHORIZATION',
```

Comment:
This looks completely fine, just wanted to raise an idea for our future selfs: Do we want to make the difference between this and the plural version a little more obvious? The current difference is only the (S), which may make things difficult when reading quickly 😅 . An example of something a little more obvious would be `SET_INDIVIDUAL_AUTHORIZATION`

Reply:
I like your suggestion 👍

Reply:
While I agree with @elazzabi's suggestion, I'm worried that would make our action names inconsistent with existing ones.